frontend/src/pages/TimeChecker.tsx

```diff
@@ -0,0 +1,3 @@
+// 시간 체크 페이지입니다.
+
+export{}
```

Comment:
```suggestion
export{}

```
파일 끝에는 빈 라인이 필요합니다 (crlf/lf 문제)